index.js

```diff
@@ -4,24 +4,28 @@ const prog = require('caporal');
 const pkg = require('./package');
 const { downloadRepo } = require('./lib/repository');
 const { getLocalDependencies } = require('./lib/local');
-const { FROM, ERROR, OK } = require('./lib/constants');
+const { ERROR, OK, REPO, SPACER } = require('./lib/constants');
 
 function init(repo, logger) {
-    downloadRepo(repo).then((files) => {
-        files.forEach(file => logger.info(OK, `${file} ${FROM} ${repo}`));
-    }).catch(err => logger.error(ERROR, err));
+    downloadRepo(repo)
+        .then((arr) => arr.map((file) => `${SPACER} - ${file}`))
+        .then((arr) => [REPO(repo)].concat(arr, ''))
+        .then((arr) => logger.info(OK, arr.join('\n')))
```

Comment:
Looking at this is a bit confusing at first.

A simpler approach would just be to log the title separately from the files.

```js
logger.info('Downloaded: vutran/editorconfig'); // formatted with colors, spacers, etc.
logger.info(arr); // I believe Caporal should reformat arrays of literals as <ul>, can `map` before if needed
```